src/component/TipDisplay.spec.tsx

```diff
@@ -66,6 +66,22 @@ describe('TipDisplay component', () => {
     expect(hoverMessage.textContent).toContain('icons');
     expect(hoverMessage.textContent).toContain('quantities');
   });
+  test('should show only one tooltip at a time on mobile', () => {
+    const mockMobile = jest.fn();
+    mockMobile.mockReturnValue(true);
+
+    MobileCheck.isMobile = mockMobile;
+
+    const { getByAltText, getAllByText } = render(<TipDisplay tip={11000 * 2} />);
+    const mediumBellBagIcon = getByAltText('Medium Bell Bag Icon');
+    const smallBellBagIcon = getByAltText('Small Bell Bag Icon');
+
+    fireEvent.click(mediumBellBagIcon);
```

Comment:
Should we also check that there's only one tooltip after clicking the first item as well? What if the user clicks the first item and it doesn't trigger, then clicks the second one and it triggers?